packages/chess_com_api/chess_com_api.1.0.2/opam

```diff
@@ -0,0 +1,44 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Chess.com API client for chess games"
+description:
+  "A client library for the Chess.com API that provides easy access to chess games, player data, tournament information, and daily puzzles. Supports fetching games in PGN format and extracting metadata."
+maintainer: ["ckaf borisnovoselov602@gmail.com"]
+authors: ["ckaf borisnovoselov602@gmail.com"]
+license: "MIT"
+tags: ["chess" "chess.com" "api" "http" "client" "games" "pgn"]
+homepage: "https://github.com/ckaf/pgn_parser"
+doc: "https://github.com/ckaf/pgn_parser"
+bug-reports: "https://github.com/ckaf/pgn_parser/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "3.19"}
+  "cohttp-lwt-unix" {>= "5.0"}
+  "yojson" {>= "1.7"}
+  "lwt" {>= "5.6"}
+  "lwt_ppx" {>= "2.0"}
+  "qcheck" {>= "0.21"}
```

Comment:
```suggestion
  "qcheck" {>= "0.21" & with-test}
```
Happy to see a QCheck user! :smiley: 
I suspect that this is only a dependency when users want to run the package tests though (`opam install ... --with-test`).
It is not a big deal (or a blocker) though. I noticed and thought I would point it out :shrug: